src/sentry/coreapi.py

```diff
@@ -161,7 +163,7 @@ def ensure_does_not_have_ip(self, data):
         if sdk:
             sdk.pop('client_ip', None)
 
-    def insert_data_to_database(self, data, start_time=None,
+    def insert_data_to_database(self, project, data, start_time=None,
```

Comment:
This function isn't exposed anywhere for code outside this repo to use right?

Reply:
Being Python and all I can't really guarantee that, but I don't see any usages in `getsentry` or `sentry-plugins` so I guess 🤞 ? 😬